docs/concepts/what-is-a-lokomotive-component.md

```diff
@@ -0,0 +1,152 @@
+
+Lokomotive also provides [rook](../component-reference/components/rook.md) and
+[rook-ceph](../component-reference/components/rook-ceph.md) components for using Rook as the storage
+solution for Lokomotive cluster.
+
+### Load Balancing/Ingress (for Packet platform only)
+
+Lokomotive provides [MetalLB](../component-reference/components/metallb.md) component for load
+balancing and [Contour](../component-reference/components/contour.md) component for ingress.
+
+Lokomotive also provides an [external-dns](../component-reference/components/external-dns.md)
+component for automatic management of DNS entries for Ingress resources.
+
+### Node management
+
+Lokomotive provides
+[calico-hostendpoint-controller](../component-reference/components/calico-hostendpoint-controller.md)
+component for adding and removing [Calico HostEndpoint
+objects]((https://docs.projectcalico.org/v3.8/reference/resources/hostendpoint)) and
+[cluster-autoscaler](../component-reference/components/cluster-autoscaler) component for adjusting
+the size of Lokomotive cluster.
+
+### Update
+
+Lokomotive provides
+[flatcar-linux-update-operator](../component-reference/components/flatcar-linux-update-operator.md)
+component for draining a node before rebooting after a Flatcar Container Linux OS update.
+
+## Listing available components
+
+To list available Lokomotive components, run the following command:
+
+```console
+$ lokoctl component list
+
+Available components:
+	 calico-hostendpoint-controller
+	 cert-manager
+	 cluster-autoscaler
+	 contour
+	 dex
+	 external-dns
+	 flatcar-linux-update-operator
+	 gangway
+	 httpbin
+	 metallb
+	 metrics-server
+	 openebs-operator
+	 openebs-storage-class
+	 prometheus-operator
+	 rook
+	 rook-ceph
+	 velero
+```
+
+## Installing a component
+
+To install a Lokomotive component, run the following command:
+
+```console
+lokoctl component install <component_name>
+```
+
+This will take the `kubeconfig` from your cluster asset directory if you run it in the folder of
+your cluster configuration files. Otherwise, the default kubeconfig location will be used
+(`~/.kube/config`) unless you specify it through the `--kubeconfig` flag or the `KUBECONFIG`
+environment variable.
+
+A set of components to install may also be provided in a `.lokocfg` file:
+
+```hcl
+component "metallb" {...}
+
+component "contour" {}
+```
+
+Specifying components in a `.lokocfg` file also allows passing configuration arguments to components
+which support them. See the [documentation](../configuration-guides/components) for individual
+components for information about the supported arguments.
+
+To install all the components listed in a `.lokocfg` file, omit the component name:
+
+```console
+lokoctl component install
+```
+
+>NOTE: `lokoctl` automatically detects all `.lokocfg` files in the working directory. This can be
+>used to organize component configuration in separate files.
+
+## Rendering component manifests
```

Comment:
In #51 I changed this section to show how to render manifests only since now that we use helm the other commands are kinda tricky and we shouldn't mention them to users. I'd like to keep it this way.